src/views/Home.jsx

```diff
@@ -1,12 +1,41 @@
 import './Home.css';
 import { Link } from 'react-router-dom';
+import React, { useState } from 'react';
+
+export function Home({ handleCreateList, handleJoinList, errorMsg }) {
+	const [token, setToken] = useState('');
+
+	const handleChange = (event) => {
+		setToken(event.target.value);
+	};
+	function handleSubmit(e) {
+		e.preventDefault();
+		handleJoinList(token);
+	}
 
-export function Home({ handleCreateList }) {
 	return (
 		<div className="Home">
 			<Link to="/list">
 				<button onClick={handleCreateList}>Create List</button>
 			</Link>
+			<div className="container">
+				<p>Join an existing shopping list by entering a three word token.</p>
+				<form onSubmit={handleSubmit}>
+					<label htmlFor="listToken">Enter List Token:</label>
+					<br />
+					<input
+						id="listToken"
+						type="text"
+						name="listToken"
+						onChange={handleChange}
```

Comment:
since the callback is so simple you could just do: 
```suggestion
						onChange={e=>setToken(e.target.value)}
```
It's not wrong of course, just suggesting an alternative option.

Reply:
Thanks for the feedback!